eng/pipelines/libraries/run-test-job.yml

```diff
@@ -55,6 +54,14 @@ jobs:
       ${{ if eq(parameters.interpreter, 'true') }}:
         testDisplayName: ${{ parameters.runtimeFlavor }}_interpreter_${{ parameters.liveRuntimeBuildConfig }}
 
+      # To run the tests we just send to helix and wait, use ubuntu hosted pools for faster providing and to not back up our build pools
+      ${{ if startsWith(parameters.pool.queue, 'BuildPool.Ubuntu') }}:
+        pool:
+          vmImage: 'ubuntu-latest'
+      
+      ${{ if not(startsWith(parameters.pool.queue, 'BuildPool.Ubuntu')) }}:
```

Comment:
Any reason to not do the same with non-Ubuntu pools? Seems a waste. Also how much of the build does this job download? We were hitting the 10gb+ limit on the publishing job, just want to make sure we are not getting close to that when doing this.

Reply:
MacOS already use hosted pools. I decided not to move windows to hosted pools to be careful about hitting the 200 parallel jobs on hosted pools across the org. We don't have as many windows legs as we do ubuntu pool based legs, so that's why I kept windows out. My first commit did include Windows, but I removed it after thinking about that.

Let me know if you think otherwise and we can include Windows here.